packages/eyevinn-skin/src/components/buttons/volume/VolumeControls.tsx

```diff
@@ -9,17 +10,27 @@ export default function VolumeControls({
   onSliderInput,
   volume,
 }) {
+  const [isMobile, setIsMobile] = useState(false);
```

Comment:
Wasn't completely sure whether to use state or ref.

Reply:
In practise this should not change during a session, right?
So it can rather just be a constant being set from the check?